### PR TITLE
refactor nifti masker family

### LINKED ISSUE #425: Refactor nifti masker code
Since we are at it, how about refactoring all the nifti masker classes into a single python file?

More concretely, how about unifying the code of...
```python
- base_masker.py
- nifti_masker.py
- nifti_region.py
- multi_nifti_masker.py
```

### PR DESCRIPTION
**Pull request by banilo**
tries to fix #425

### CODE REVIEW

File: nilearn/input_data/nifti_labels_masker.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+from .._utils import logger
+from .._utils import CacheMixin
+from .._utils.niimg_conversions import _check_same_fov
+from .. import signal
+from .. import region
+from .. import masking
+from .. import image
+
+
+def _compose_err_msg(msg, **kwargs):
```
Comment (AlexandreAbraham) [inline nilearn/input_data/nifti_labels_masker.py]: Duplicated code.

Comment (banilo) [inline nilearn/input_data/nifti_labels_masker.py]: I know, there is a similar situation with a helper function in the tests. What's your suggestion to handle this?

Comment (AlexandreAbraham) [inline nilearn/input_data/nifti_labels_masker.py]: My suggestion is: not duplicating the code.

What we can do is to let it pass in this PR. Given that it's a 5 line private function, it's not a big matter. Afterward, I suggest that we take all the little tricks that we use to have a nice display and put them somewhere in _utils.

**Comment by banilo** [comment]
rebased!

**Pull request merged**